libraries/AP_HAL_ChibiOS/hwdef/revo-mini-i2c/hwdef.dat

```diff
@@ -148,8 +149,14 @@ define HAL_PROBE_EXTERNAL_I2C_COMPASSES
 # SPI devices
 SPIDEV mpu6000   SPI1 DEVID1 MPU_CS   MODE3 1*MHZ 8*MHZ 
 SPIDEV dataflash SPI3 DEVID1 FLASH_CS MODE3 32*MHZ 32*MHZ
+SPIDEV sdcard    SPI3 DEVID2 SDCARD_CS MODE0 400*KHZ 25*MHZ
 
 # enable logging to dataflash
 define HAL_LOGGING_DATAFLASH_ENABLED 1
 
+# filesystem setup on sdcard
+define HAL_OS_FATFS_IO 1
```

Comment:
I don't think you can do this - this will make the default logging sdcard rather than dataflash. I think you would need to do this as a new hwdef.dat

Reply:
Ok, that's something I was wondering about